Makefile

```diff
@@ -158,10 +159,10 @@ dev-down-kind:
 	cd install/scripts && ./kind-down.sh
 
 .PHONY: dev-skaffold-gcp
-dev-skaffold-gcp: export PROJECT_ID=$(shell gcloud config get project)
+dev-skaffold-gcp: PROJECT_ID ?=$(shell gcloud config get project)
```

Comment:
Is this needed here? PROJECT_ID is set at top level now

Reply:
it's not set at the top level and it shouldn't be because then it gets executed too many times and would fail if you're not on GCP?

Reply:
Mis-read, looks good